en/mdn_plus_privacy.md

```diff
@@ -13,15 +13,15 @@ This privacy notice explains what data MDN Plus collects, shares, and why. We al
 
 ### Information we collect {: #InfoWeCollect }
 
-__Firefox Account information.__ This service requires a Firefox Account, which provides MDN Plus read-only access to your Firefox Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Firefox Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
+__Mozilla Account information.__ This service requires a Mozilla Account, which provides MDN Plus read-only access to your Mozilla Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Mozilla Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
```

Comment:
Should be lower case `a` in `Mozilla account`. We used to call it Firefox Account but later it changed to the lower `a`.  Make sure to update the entire document accordingly.

Reply:
Sorry! Done (I think)